examples/map.asl

```diff
@@ -27,11 +28,15 @@
         "States": {
           "Generate UUID": {
             "Type": "Pass",
-            "End": true,
+            "Next": "Sleep",
             "Parameters": {
               "uuid.$": "States.UUID()"
-            },
-            "OutputPath": "$.uuid"
+            }
+          },
+          "Sleep": {
+            "Type": "Task",
+            "Resource": "docker://docker.io/agrare/sleep:latest",
+            "End": true
```

Comment:
NOTE this gives us a sub-iteration that runs a task that doesn't complete immediately like a Pass state would so it exercises the Workflow.wait event-handling code for child-tasks